Corona-Warn-App/src/main/java/de/rki/coronawarnapp/diagnosiskeys/download/DownloadDiagnosisKeysTask.kt

```diff
@@ -82,18 +81,15 @@ class DownloadDiagnosisKeysTask @Inject constructor(
                 return object : Task.Result {}
             }
 
-            // EXPOSUREAPP-3878 just to be sure that risk level calculation can take place
-            if (TimeVariables.getLastTimeDiagnosisKeysFromServerFetch() != null) {
```

Comment:
why was the additional condition removed? it doesn't do any harm, does it? but it assures that there is actually data for the risk task